lib/hashes.md

```diff
@@ -5,40 +5,47 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a Hash?
 
-Replace me with your response...
+An unordered collection of unique keys and their values.
 
 # What are some examples of information that would be Hashes as opposed to some other data type?
 
-Replace me with your response...
+lists that require to be broken out into subcategoires that do not require a particular order as opposed to just lists that fall into one category.  I.E. 
+
+student_ages = {
+    "Jack" => 10,
+    "Jill" => 12,
+    "Bob" => 16
+}
+
 
 # How are Hashes and Arrays similar? How are they different?
 
-Replace me with your response...
+Arrays allow you to make an ordered list.  Hashes allow you to make an unordered list (keys) but with details/definitions (values).
 
 # How do you retrieve a particular value from a Hash?
 
-Replace me with your response...
+Write the name of the object and follow it with square brackets [] and inside the brackets write the key. If the key is a string use quotation marks.
 
 # How do you add information to a Hash?
 
-Replace me with your response...
-
+hash.merge! As far as I can tell it doesnt alter the original hash but creates a new one using the previous hash and the new hash.  It took me 2 hours to find this answer...
+hash = {"a" => 1} hash.merge!("b" => 2, "c" => 3)
```

Comment:
True, merging can be used to add information to a Hash. It's a fairly "heavy" operation for such a mundane task though. So you might like to know that just as Arrays let you `push` new elements, so too do Hashes have an easier method for this. Check this [google search](https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=ruby+hash+push+new+element) and let me know if you'd like further guidance.